zebra-rpc/src/methods/tests/vectors.rs

```diff
@@ -288,6 +289,7 @@ async fn rpc_getrawtransaction() {
                         id: UnminedTxId::Legacy(tx.hash()),
                         transaction: tx.clone(),
                         size: 0,
+                        conventional_fee: Amount::zero(),
```

Comment:
I think existing tests are sufficient but I'll leaving this here to give @dconnolly a chance to disagree.